### ISSUE TITLE
Addition of materials

### ISSUE THREAD

**Issue by Samuel11s** [Samuel11s/My-first-tutorial-project#2]
# Step 1
Addition of materials that I would have to use in this project.

**Comment by AshDest** [comment]
Please @Samuel11s , Can you give more details on what you were able to add and their usefulness?

**Comment by Samuel11s** [comment]
Yes @AshDest, by the way I just added an image serving as a background in this project, but you can add one too or enrich this project with your ideas too.
